### ISSUE TITLE
Issue with 'TRACY_DELAYED_INIT'

### ISSUE THREAD

**Issue by lispbub** [wolfpld/tracy#43]
Hi there,

I had an issue with combining 'TRACY_DELAYED_INIT' on Windows, causing fails with rpmalloc.
It might be also correlated to issue #42, since (as far as I understand) for apple the 'TRACY_DELAYED_INIT' is set.

During investigation, I found that line 163 in TracySystem.cpp caused all the fuss:

````
        const auto sz = strlen( name );
        char* buf = (char*)tracy_malloc( sz+1 );
        memcpy( buf, name, sz );
        buf[sz+1] = '\0';
````
``  buf[sz+1] = '\0';`` should be ``  buf[sz] = '\0';`` I guess.

And thanks for that great  software :D

**Comment by wolfpld** [comment]
Ah crap. Thanks!

**Comment by wolfpld** [comment]
Fixed in 0744355a.
